src/libs/ReportUtils.ts

```diff
@@ -4248,6 +4260,7 @@ function buildOptimisticTaskCommentReportAction(
 
     // These parameters are not saved on the reportAction, but are used to display the task in the UI
     // Added when we fetch the reportActions on a report
+    // eslint-disable-next-line deprecation/deprecation
```

Comment:
Hmm it doesn't seem like this rule exists in eslint, also it doesn't look like eslint is complaining about this line at all, but it's rather TS

<img width="668" alt="Screenshot 2024-09-19 at 22 34 40" src="https://github.com/user-attachments/assets/1d1f75cc-61b1-406e-931b-fa542a8baed0">

Reply:
Hm so it's actually interesting. Locally, it says that the error is deprecation/deprecation when I run `npm run lint-changed`
<img width="1180" alt="image" src="https://github.com/user-attachments/assets/5a44bc6e-3175-4105-9d4c-64198494d664">

I'm going to switch it to `eslint-disable-next-line` for now to suppress the error? What do you think? The other thing we can do is what was done [here](https://github.com/Expensify/App/pull/49324/files#diff-577fcc5f3a5e4930916ff310e61c38edfa093792fe3ab55b9a30d2aecd8811dbR4240) with a spread operator but I'm not sure if it's worth it.